develop/dev-guide-sample-application-python-django.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Connect to TiDB with Django
+summary: Learn how to build a Django application with TiDB. This tutorial shows Python sample code snippets that work with TiDB using Django.
+---
+
+# Build a Django App with TiDB
+
+TiDB is a MySQL-compatible database, and [Django](https://www.djangoproject.com/) is a popular web framework for Python.
+
+In this tutorial, you can learn how to use TiDB and Django to accomplish the following tasks:
+
+- Set up your environment.
+- Connect to your TiDB cluster using Django.
+- Build and run your application. Optionally, you can find sample code snippets for basic CRUD operations.
+
+> **Note:**
+>
+> This tutorial works with TiDB Serverless, TiDB Dedicated, and TiDB Self-Hosted.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- [Python 3.8 or higher](https://www.python.org/downloads/).
+- [Git](https://git-scm.com/downloads).
+- A TiDB cluster.
+
+<CustomContent platform="tidb">
```

Comment:
what are the differences between platform="tidb" and "tidb-cloud"? Seems like the same content.

Reply:
This document will exists in both TiDB Cloud and TiDB, but the following documentation is only available in TiDB, so there are some differences in link address: 
- https://docs.pingcap.com/tidb/stable/quick-start-with-tidb#deploy-a-local-test-cluster
- /quick-start-with-tidb.md#deploy-a-local-test-cluster